CITATION.cff

```diff
@@ -0,0 +1,19 @@
+cff-version: 1.2.0
+message: "If you are referring to the STAC specification in your publications, please cite it as below."
+type: software
```

Comment:
Is it a software or a standard?

Reply:
This is a requirement due to citation-file-format/citation-file-format#403